### ISSUE TITLE
Slower performance opening action dialog in dark theme

### ISSUE THREAD

**Issue by AshleyScirra** [Scirra/Construct-3-bugs#3427]
## Problem description

Clicking the 'Add action' button in an event sheet can be slower in the dark theme than the default theme.

## Attach a .c3p

[Large event sheet test.zip](https://github.com/Scirra/Construct-3-bugs/files/3750266/Large.event.sheet.test.zip)

## Steps to reproduce

1. Expand Group1 and scroll near to end 
2. Click 'Add action' dialog (note be careful not to move mouse and change cursor causing #3426, which is a separate issue)

Compare performance in default and dark themes with profile running.

## Observed result

Tested Canary 80.0.3946.0.
Default theme: ~150ms
Dark theme: ~1.5s

## Expected result

Dark theme should be as fast as default theme

## More details
A related case is then clicking 'Next'. This opens the next dialog, which is probably slow for the same reason.

It may well be another Chrome bug doing layout when it doesn't need to. We have no tools to investigate why there is a difference here, so this will be tough.

**Comment by AshleyScirra** [comment]
Eventually tracked this down by trial and error and fixed a wayward selector that seemed to be extremely slow. Should be a lot faster in the next release.

**Comment by SnipG** [comment]
I did some quick check on latest eventsheet perf improvements.
Not sure if you already changed something, but 172 vs 171. When action is opened with mouse and expression dictonary also opens, then there is somesort of small lag in r172+ and onwards.
I did not test 171.2 and 171.3, just 171 vs 172.

There is clearly something happening, not sure what. I tried like 4 times 171, then 172.

Example i used. 
[SmallTest.zip](https://github.com/Scirra/Construct-3-bugs/files/3787694/SmallTest.zip)
